helix-view/src/editor.rs

```diff
@@ -1308,6 +1308,10 @@ impl Editor {
             || self.config().popup_border == PopupBorderConfig::Menu
     }
 
+    pub fn should_extend(&self) -> bool {
+        self.mode == Mode::Select
```

Comment:
Let's inline this function, it's only used in two spots and it adds more API surface area to the Editor

Reply:
done